frontend21/scss/amp-dev.scss

```diff
@@ -45,6 +45,7 @@
 @import 'components/content/teaser.scss';
 @import 'components/content/intro.scss';
 @import 'components/extra/about/components-showcase.scss';
+@import 'components/newsletter.scss';
```

Comment:
Can you move this to a folder named `extra/home` as this is a component solely used on the homepage?

Reply:
Yes, that makes total sense. Thanks.